### DISCUSSION TITLE
installed packages

### DISCUSSION THREAD

**Discussion by msMatix** [Morganamilo/paru#994] [Ideas]
Is there a file where you can Store the installed aur packages ? 
And if you call the remove Command the Package will be deleted from the file ? Just like Poetry in Python? And at the end you Just say aur install and all the packages from the list will be installed ? This List could also be tracked via a github repo?

Greetings 
Matix ;)

**Comment by x70b1** [comment]
This and much more can be done with [aconfmgr](https://github.com/CyberShadow/aconfmgr).
But its more complex and need time for setup.

**Reply by msMatix** [reply]
ah thanks !

**Comment by Morganamilo** [comment]
You can just keep your list in a file and `paru -S --needed - < file` at will if all you want is to just install a list of packages. There's other more complex solutions for more advanced use cases using separate tools.
